include/cucumber-cpp/internal/step/StepManager.hpp

```diff
@@ -157,7 +162,7 @@ class StepManager {
 };
 
 
-static std::string toSourceString(const char *filePath, const int line) {
+static inline std::string toSourceString(const char *filePath, const int line) {
```

Comment:
what kind of performance improvement do you expect by inlining this? or was it done for a different reason?